api.py

```
@@ -145,22 +218,56 @@ async def _validate_packs_async(token, user_id):
-    raw_token = os.environ.get("TELEGRAM_BOT_TOKEN", "")
-    token_match = re.search(r'\d+:[A-Za-z0-9_-]{35,}', raw_token)
-    if token_match:
+    # Only validate against Telegram if refresh parameter is provided
+    refresh = request.args.get("refresh", "").lower() in ("true", "1", "yes")
+    if refresh and SETTINGS.telegram_bot_token:
         try:
-            packs = _run_async(_validate_packs_async(token_match.group(0), uid))
+            packs = _run_async(_validate_packs_async(SETTINGS.telegram_bot_token, uid))
             return ok(packs)
         except Exception:
             pass
```

Ruff (0.15.7)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)